lib/ticks-to-tree.js

```diff
@@ -15,19 +19,22 @@ function hasFileLocation (jsFrameName) {
   return /:\d+:\d+$/.test(jsFrameName)
 }
 
-function ticksToTree (ticks, options = {}) {
-  const { mapFrames, inlined, pathToNodeBinary = process.execPath } = options
-  const merged = {
-    name: 'all stacks',
-    value: 0,
-    top: 0,
-    children: []
-  }
-  const unmerged = {
-    name: 'all stacks',
-    value: 0,
-    top: 0,
-    children: []
+async function ticksToTree (ticks, options = {}) {
```

Comment:
Is it necessary to turn this into an `async` function?

Reply:
Not sure. I'm picking up this from previous PR changes. I don't see why it is like this currently.

Reply:
Ah, it uses await here:
`if (sourceMaps) {
    await Promise.all(Object.keys(jsLibraries).map(async (jsLibrary) => {
      return loadSourceMap(jsLibrary, sourceMaps)
    }))
  }`
I don't have an answer to your question really. I would likely need the original author to chime in. @pimlie ?